src/Action/CommentListAction.php

```diff
@@ -15,10 +15,10 @@
 
 use Sonata\NewsBundle\Model\CommentInterface;
 use Sonata\NewsBundle\Model\CommentManagerInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 
-final class CommentListAction extends Controller
+final class CommentListAction extends AbstractController
```

Comment:
It is correct but: 
https://symfony.com/doc/4.4/best_practices.html#make-your-controller-extend-the-abstractcontroller-base-controller

Can you drop extenging `*Controller` here, inject require service and fix code?